packages/react-vapor/tsconfig.build.json

```diff
@@ -1,13 +1,14 @@
 {
-    "extends": "./tsconfig",
+    "extends": "./tsconfig.json",

```

Comment:
Do you really need the extensions?

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends

Reply:
I don't think so, it is what was used in the example I was following https://github.com/RyanCavanaugh/learn-a